target/linux/ath79/dts/ar7161_netgear_wndr.dtsi

```diff
@@ -222,7 +225,30 @@
 
 	pll-data = <0x11110000 0x00001099 0x00991099>;
 
-	mtd-mac-address = <&art 0x06>;
+	nvmem-cells = <&macaddr_art_06>;
+	nvmem-cell-names = "mac-address";
 
 	phy-handle = <&phy4>;
 };
+
+&art {
+	compatible = "nvmem-cells";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	macaddr_art_0: macaddr@0 {
+		reg = <0x0 0x6>;
+	};
+
+	macaddr_art_00: macaddr@00 {
```

Comment:
It looks like this is here twice.

Reply:
Nice find! any idea why there is 0x0 and 0x00 ? Nobody notice that?

Reply:
> any idea why there is 0x0 and 0x00 ?

No special reason. Looks like I have used different notation in b58df9919 (originally 25ea21d7) and 437e35f8 

WNDR3700v1,v2 and 3800 has three MAC stored there.
0x00 is used for eth0 and wlan0,
0x06 is used for eth1
0x0c is used for wlan1

The representation of 0 differs between lines 156 and 212. 0x0 vs 0x00

Reply:
i'm updating the script to use the 0x0 notation and I will push fixed dts. Think this problem is only in ath79, am i wrong?

Reply:
There might be lots of variation if you look closer. Especially the ar71xx/ath79 device conversions have been made with so many people, and partially by modifying/copying from various other files, that there may well be other similar things. (also between DTSI and DTS).

Reply:
Assuming 0x00 variants are only present in ath79, i should have fixed all the dts... (there were 9)